### DISCUSSION TITLE
Mark as answer button is confusing vs browser

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44612] [Mobile] [answered]
GitHub Mobile beta

The check here is Mark as answer:
![Screenshot_20230117-160912.png](https://user-images.githubusercontent.com/2119212/213013064-825d7f31-8324-4d6d-a787-1cd7f2732889.png)

In a browser, the check is just a check and there's a separate button for Mark as answer:
![Screenshot_20230117-161046.png](https://user-images.githubusercontent.com/2119212/213013251-bee454c4-9d0c-4193-b260-ab016a3589c5.png)

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref I understand. We chose this to try to save real estate on the screen. `Mark as answer` takes up a lot of space. Thanks for the feedback though!

**Reply by jsoref** [reply]
Could you talk to the web team and ask them to merge the two widgets while an item isn't marked as the answer?

`(✔️ Mark as answer)` would be similar to `(↑1)` that'd at least teach web users that it's the ✔️ that they should be looking for.
